brickflow/engine/workflow.py

```diff
@@ -379,6 +384,7 @@ def notebook_task(
             task_type=TaskType.NOTEBOOK_TASK,
             task_settings=task_settings,
             depends_on=depends_on,
+            if_else_outcome=if_else_outcome,
```

Comment:
What is the need for adding this in all the tasks?

Reply:
Other tasks may depends on condition task right, For example if my condition task returned "true" then run my RunJob Task otherwise run some notebook task with trigger mails.

Reply:
<img width="1267" alt="image" src="https://github.com/Nike-Inc/brickflow/assets/57700376/56da11fe-d4ff-418f-bb99-d6fe19ffe1d1">

Reply:
![image](https://github.com/Nike-Inc/brickflow/assets/57700376/239e907f-c45f-476f-97a7-1dc19cf8b9ed)
added screenshot for reference, SQL task is dependent on if/else task. If if/else task returns true then will run SQL task.